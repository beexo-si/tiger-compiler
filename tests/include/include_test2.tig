let
    include "dec_test.tigd"
    var a := randomfunction(5)
in
    0
end
