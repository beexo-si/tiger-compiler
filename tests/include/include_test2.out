Fail: Error -- l√≠nea 3: Funcion inexistente (randomfunction)
